pages/user/signin.tsx

```diff
@@ -45,10 +48,31 @@ function SignIn() {
   );
 }
 
-export async function getServerSideProps({ query }: GetServerSidePropsContext) {
-  const loginStatus = query.status ? true : '';
+export async function getServerSideProps({
+  req,
+  res,
+  query,
+}: GetServerSidePropsContext) {
+  const statusQuery = query.status as string;
+  const authArr = statusQuery?.split(/[?=]/);
+
+  const status = authArr && authArr[0];
+  const access = authArr && authArr[2];
+  const refresh = authArr && authArr[4];
+
+  setCookies('access', access, { req, res });
+  setCookies('refresh', refresh, { req, res });
+
+  if (status === 'signin') {
```

Comment:
```suggestion
  if (status === 'signup') {
```

회원가입이라 signup을 쓰는게 좋을 것 같아여!

Reply:
이 pr 에서 변경하면 다른 브랜치 컨플릭나서 이번에 연관된 pr 다 머지하고 고치는게 편할 것 같슴당!

Reply:
오 알겠습니다 병현님 리뷰 감사합니다! 제가 놓친 상세한 부분까지 봐주셔서 넘 감사합니다!! 👍